### DISCUSSION TITLE
Lost my whole project after created a repository from codespace project!

### DISCUSSION THREAD

**Discussion by Faheem2k19** [community/community#44417] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I created a react project from a blank template of codespace. When I completed it, I created a repo directly from that codespace project. After creating the repo, I noticed my project wasn't committed. And then, I quickly opened the project from codespace, guess what?? Apart from the node_modules folder all files are gone. I don't know what to do now.

The repo link is here- https://github.com/Faheem2k19/agency-xyz

It's all empty!!!!

**Comment by salvador-barboza** [comment]
Hi @Faheem2k19, sorry to hear you are experiencing issues. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

Looking at your repo, it seems that `/agency-xyz` is a Git submodule. I'd suggest to open your original Codespace, and copy the content from `/workspaces/codespaces-blank/agency-xyz` to `/workspaces/codespaces-blank`, then commit and push the changes. After that, everything will be there.

 When exporting a Codespace, a new Git repo will be initialized in `/workspaces/[repo-name]` e.g. `/workspaces/codespaces-blank`. A [Git repo will not track submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules). This is why `/workspaces/codespaces-blank/agency-xyz` was not included on https://github.com/Faheem2k19/agency-xyz.

**Comment by jangala-laxman** [comment]
Even Im facing the same problem after uploading my repository to github via codespaces, I lost it somehow. And even it is not present in my repositories as well.
